torch/csrc/jit/script/compiler.cpp

```diff
@@ -947,13 +984,6 @@ struct to_ir {
         case TK_AUG_ASSIGN:
           emitAugAssignment(AugAssign(stmt));
           break;
-        case TK_GLOBAL:
```

Comment:
this code was totally bogus stuff left over from caffe2 script!

Reply:
![image](https://user-images.githubusercontent.com/4685384/63557116-331f4200-c4fc-11e9-996a-81011b095d95.png)